docs/content/guides/rows/row-sorting.md

```diff
@@ -1,79 +1,141 @@
 ---
 id: 6o0zftmc
-title: Row sorting
-metaTitle: Row sorting - JavaScript Data Grid | Handsontable
-description: Sort your data in ascending, descending, or a custom order, across one or multiple rows of the data grid.
+title: Rows sorting
+metaTitle: Rows sorting - JavaScript Data Grid | Handsontable
+description: Sort rows alphabetically or numerically, in ascending, descending or a custom order, across one or multiple columns.
 permalink: /row-sorting
 canonicalUrl: /row-sorting
+tags:
+  - row sorting
+  - column sorting
+  - columns sorting
+  - columnSorting
+  - multicolumn sorting
+  - multi-column sorting
+  - multiColumnSorting
 react:
   id: h4jfevxj
-  metaTitle: Row sorting - React Data Grid | Handsontable
+  metaTitle: Rows sorting - React Data Grid | Handsontable
 searchCategory: Guides
 ---
 
-# Row sorting
+# Rows sorting
 
-Sort your data in ascending, descending, or a custom order, across one or multiple rows of the grid.
+<p style="font-size:20px;">Sort rows alphabetically or numerically, in ascending, descending or a custom order, across one or multiple columns.</p>
```

Comment:
Let's remove the inline style and restore the "Overview" heading with a more extensive description, like so:

> ## Overview 
> Handsontable's sorting feature allows you to easily rearrange rows of data based on the values in specified columns. You can sort rows alphabetically, numerically, or based on the data type set in the configuration. You can choose to sort rows in ascending or descending order or define a custom order to suit your needs. This feature is especially useful for analyzing and organizing large datasets, helping you to identify patterns and trends.

The text above might be adjusted however you wish, just keep it more descriptive than it was.

Reply:
How about this? [c72bac8](https://github.com/handsontable/handsontable/pull/10183/commits/c72bac8f8c65bacf61c825482506545723214003)

https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/a/allow-allows
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/l/let-lets